shared/js/browser/ios-communication.js

```diff
@@ -86,7 +86,6 @@ export function privacyDashboardTelemetrySpan(args) {
  */
 async function fetch(message) {
     if (message instanceof CheckBrokenSiteReportHandledMessage) {
-        privacyDashboardShowReportBrokenSite({});
```

Comment:
On iOS, this method was only being used to fire the same pixel we’re now firing when rendering the breakage form